lib/generators/language_alias_generator.rb

```diff
@@ -0,0 +1,20 @@
+class LanguageAliasGenerator
+
+  def alias_class_method(class_name, orig_name, alias_name)
+    klass = Object.class_eval(class_name)
+    singleton_klass = klass.singleton_class
+
+    singleton_klass.send(:alias_method, alias_name, orig_name)
+  rescue StandardError => e
+    puts e.message
+  end
+
+  def alias_instance_method(class_name, orig_name, alias_name)
+    klass = Object.class_eval(class_name)
+
+    klass.send(:alias_method, alias_name, orig_name)
+  rescue StandardError => e
+    puts e.message
```

Comment:
Do we benefit from capturing the error and simply putting the message instead?

Reply:
I am not aware of what kind of error it throws which was meant to be captured by StandardError. The original code expects an error.
Basically having Custom error classes helps in debugging in logs. In this case I didn't knew what type of error we're expecting so I added puts.
We can skip rescuing the error as well, although the error message will not be obvious. Do you want me to remove it ?